styles.css

```diff
@@ -544,3 +548,291 @@ footer .logo #logo #logoSpan,
+    width: 100%;
+  }
+
+  .about-camp-logo {
+    position: relative;
+    top: 120px;
+    padding: 40px 280px;
+  }
+
+  .past-camp-section {
+    position: relative;
+    top: 120px;
+    padding: 40px 200px;
+  }
+
+  .past-camp-section .camps-container {
+    flex-direction: row;
+  }
+
+  .headline-section .headline-p {
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .headline-section .headline-date {
+    color: #272a31;
+    font-size: 28px;
+    font-weight: 800;
+    line-height: 32px;
+  }
+
+  .headline-section .h-address {
+    font-weight: 400;
+  }
+
+  /* This styles the program section */
+  .program-section {
+    top: 120px;
+    padding: 50px 120px;
+  }
+
+  .program-section .text-container h3 {
+    font-size: 44px;
+  }
+
+  .program-section .program-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 5px;
+  }
+
+  .program-section .program-container .program-list {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    box-shadow: none;
+    gap: 15px;
+    padding: 10px;
+    transition: all 0.3s ease-in-out;
+  }
+
+  .program-section .program-container .program-list:hover {
+    /* border: 2px solid #d3d3d3bc; */
+    box-shadow: 0 1px 8px #fff;
+    cursor: pointer;
+  }
+
+  .program-section .program-container .program-list img {
+    width: 100px;
+    height: 80px;
+  }
+
+  .program-section .program-container .program-list h4 {
+    width: 100%;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 32px;
+  }
+
+  .program-section .program-container .program-list p {
+    width: 100%;
+    text-align: center;
+    font-size: 12px;
+    line-height: 20px;
+  }
+
+  .program-section .join-btn-container {
+    display: none;
+  }
+
+  .program-section .join-link-container {
+    display: block;
+    margin: 50px;
+  }
+
+  /* This styles the speaker section */
+  .speaker-section {
+    position: relative;
+    top: 120px;
+    padding: 40px 120px 60px;
+  }
+
+  .speaker-section .speaker-grid {
+    display: none;
+  }
+
+  .speaker-section .speaker-grid-desk {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    gap: 0 20px;
+  }
+
+  .speaker-section .speaker-container {
+    padding: 10px;
+    box-shadow: 0 1px 8px #272a312c;
+  }
+
+  .speaker-section .speaker-container .img-container {
+    background-size: 100px 100px;
+  }
+
+  .speaker-section .text-container .speaker-desc {
+    font-size: 14px;
+    line-height: 18px;
+    font-weight: 600;
+  }
+
+  .speaker-section .more-btn-container {
+    display: none;
+  }
+
+  /* This styles the partner section */
+  .partner-section {
+    position: relative;
+    top: 120px;
+    gap: 10px;
+  }
+
+  /* This styles the footer section */
+  footer {
+    position: relative;
+    top: 120px;
+    flex-direction: row;
+    gap: 30px;
+  }
+
+  footer #footerText {
+    font-size: 12px;
```

Comment:
- [ ] Great Job adding the dark background for the about page, but it should stay white for the mobile version of the about page, here's a small summary to make it clear

Mobile Version:

- [ ] Always White, for both home & about **( kindly fix this 🙏 )**

Desktop Version

- [ ] dark for the about page **(you already implemented this ✔️ )**
- [ ] White for the home page **(you already implemented this ✔️ )**